cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -166,6 +169,25 @@ func (r *Runner) setAWSEnv() error {
 	return nil
 }
 
+func (r *Runner) setAzureEnv() error {
```

Comment:
this function never returns any error. The return type should not be `error`

Reply:
Ah I forgot to have it get called in a loop, I'll remove the error return for now and if we ever another we can refactor it to have a common api for adding the env vars.